source/playbooks/customize-a-playbook.rst

```diff
@@ -57,12 +57,13 @@ Make checklists
 Assign tasks and due dates
 --------------------------
 
-In some workflows, there are time constraints on tasks and others may have more flexible timeframes.
+In some workflows, there are time constraints on tasks and others may have more flexible timeframes. Associating tasks with deliverable dates provides visibility into workloads and helps run members be accountable during the run.
 
-Tasks can be assigned to run members, with a due date. Select the **Toggle Run Details** icon to open the **Run Details** screen. Hover over the task you’d like to edit and select the calendar icon to assign a due date.
+To assign a due date to a task, select the **Toggle Run Details** icon to open the **Run Details** screen. Hover over the task you’d like to edit and select the calendar icon to assign a due date. Due dates can be used to sort tasks in the run overview.
```

Comment:
Great question, I'll refine this.

Reply:
`Due dates can be used to sort tasks in the run overview.` - we don't have sorting functionality in the `run overview`, it seems a little confusing to me. (1/5) what if we use `prioritize` instead of 'sort':
`Due dates can be used to prioritize tasks`.